integration_test_project/src/main/cpp/Robot.cpp

```diff
@@ -23,40 +24,32 @@
 #include "SimpleMotor.h"
 #include "interface/SysIdDrivetrain.h"
 #include "interface/SysIdGeneralMechanism.h"
+#include "logging/SysIdDrivetrainLogger.h"
+#include "logging/SysIdGeneralMechanismLogger.h"
 
 class Robot : public frc::TimedRobot {
  public:
-  Robot() : frc::TimedRobot(5_ms) {
-    frc::LiveWindow::GetInstance()->DisableAllTelemetry();
-  }
+  Robot() : frc::TimedRobot(5_ms) { m_driveLogger.UpdateThreadPriority(); }
   void RobotInit() override {
     // Flush NetworkTables every loop. This ensures that robot pose and other
     // values are sent during every iteration.
     SetNetworkTablesFlushEnabled(true);
     frc::SmartDashboard::PutString("SysIdTest", "Drivetrain");
-    frc::SmartDashboard::PutNumber("SysIdAutoSpeed", 0.0);
   }
 
   void DisabledInit() override {
-    m_mechanism->SetMotor(0);
+    m_mechanism->SetMotor(units::volt_t{0});
 
-    m_driveMechanism->SetLMotor(0);
-    m_driveMechanism->SetRMotor(0);
+    m_driveMechanism->SetLMotor(units::volt_t{0});
+    m_driveMechanism->SetRMotor(units::volt_t{0});
 
     m_arm.ResetReadings();
-
-    if (m_counter > 0) {
-      wpi::outs() << "Collected " << m_counter << " data points.\n";
-
-      std::stringstream ss;
-      for (const auto& pt : m_data) {
-        ss << std::to_string(pt) << ", ";
-      }
-
-      frc::SmartDashboard::PutString("SysIdTelemetry", ss.str());
-
-      m_counter = 0;
-      m_data.clear();
+    if (m_test == "Drivetrain") {
+      std::cout << "SEND DRIVE\n";
+      m_driveLogger.SendData();
+    } else {
+      std::cout << "SEND Gneral\n";
```

Comment:
Typo (and did you mean to capitalize the entire word like in drivetrain above?)

Reply:
Oops, those were mainly debug prints